src/app.py

```diff
@@ -0,0 +1,65 @@
+from tasks import todo_list,create_task,delete_task,delete_all_tasks,mark_as_finished
+from accounts import accounts, add_account,login,user
+
+
+if __name__ =='__main__':
+
+
+    name=input("Enter your name:  ")
+    password=''
+    
+    while name in accounts:
+        login(name,password)   
+        print("{} you are logged in".format(name))     
+     
+    else:
```

Comment:
A Python `while` loop does not use `else`. Else statements are only in `if` statements.